opengapps-files.mk

```diff
@@ -1,7 +1,47 @@
 # Always needed
-PRODUCT_COPY_FILES += $(call gapps-copy-to-system,all,etc)
 PRODUCT_COPY_FILES += $(call gapps-copy-to-system,all,framework)
 
+GAPPS_NEXUS_CODENAMES += \
+    crespo \
+    crespo4g \
+    maguro \
+    toro \
+    toroplus \
+    grouper \
+    tilapia \
+    manta \
+    mako \
+    flo \
+    deb \
+    hammerhead \
+    flounder \
+    shamu \
+    angler
+
+GAPPS_PIXEL_CODENAMES += \
```

Comment:
do we need "sailfish" here, too?

Reply:
Yes. And I just noticed the TARGET_PRODUCT is named aosp_sailfish and aosp_muskie instead...so I'll have to handle that. Bleh.